code/modules/projectiles/ammunition/ballistic/rifle.dm

```diff
@@ -10,7 +10,7 @@
 /obj/item/ammo_casing/a762/enchanted
 	projectile_type = /obj/item/projectile/bullet/a762_enchanted
 
-// 5.56mm (M-90gl Carbine)
+// 5.56mm (NT-ARG Boarder)
```

Comment:
Stop renaming calibers for arbitrary reasons

Reply:
the M-90gl is based on the IRL P90. This is just making the M-90gl use the same caliber for flavor. I might use it to make a nukie exclusive sidearm to replace the stechkin using it (Five-Seven Equivalent)